updated tags, but rebased

onebeastchris opened the pull request:
some QOL tweaks; see https://github.com/GeyserMC/GeyserDiscordBot/pull/251 for the original PR with explanations

Jens-Co reviewed (approved):
Looks good to me!

onebeastchris commented:
Had to adjust a tag, see https://github.com/GeyserMC/GeyserWiki/pull/104

Konicai commented on src/main/resources/tags/links/rpconvert.tag:
```diff
@@ -4,3 +4,10 @@ type: text
+Unfortunately, this tool is not updated to support packs for versions newer than 1.17.
+
+To convert a resource pack with CustomModelData, use the following converter: https://github.com/Kas-tle/java2bedrock.sh 
+For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, espescially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```
```suggestion
For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, especially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```
Also can we make "Custom Resource Pack Help" a channel link? Its a thread so people will probably have a hard time finding it

Konicai commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
```
https://floodgate-uuid.heathmitchell1.repl.co/ works now so I would just include both

Camotoy commented on src/main/resources/tags/info/bungee.tag:
```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
 Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
+
+To properly set up Floodgate on the proxy <-> Floodgate on the backend connection, do the following:
+1. Set up player-info-forwarding in the proxy configuration files.
```
I'm pretty sure this setting is Velocity only.

onebeastchris replied:
yea, right, I should adjust it to also include the bungee equivalent.. ip-forward was it iirc

Konicai commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```
Honestly I would remove all this. There are two websites currently online for it and if they ever both go offline we can just pin a message in the support channel in the meantime

onebeastchris replied:
right, I'll include a proper link to the thread

Konicai commented on src/main/resources/tags/info/bungee.tag:
```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
+1. Set up player-info-forwarding in the proxy configuration files.
+2. Replace the Floodgate key.pem file on the backend servers (the servers behind the proxy) with the key.pem file from the proxy.
+3. Enable "send-floodgate-data" in the Floodgate config.yml on the Proxy. Be careful - this requires ALL the backend servers to have floodgate installed.
+
```
Can we just link to the wiki? I know people don't like following links but it has better resources on info forwarding for each proxy (isn't `player-info-forwarding` for velocity?)

onebeastchris replied:
breh replied to wrong one
anyways, yes, I've added a new commit that mentions that this is velocity only vs bungee..which, ironically, the wiki doesn't do (no mention of velocity)
ig I'll make a wiki pr for that and just link to the wiki in this tag

Konicai replied:
👍

Konicai commented on src/main/resources/tags/info/prefix.tag:
```diff
@@ -8,3 +8,5 @@ We strongly recommend that you **do not remove** the prefix, but instead change
 
 You can change the prefix (`.`) in players' names by editing the option in Floodgate's config.yml file. Characters that a Java player cannot normally choose to be part of their names are recommended.
 Some examples include `+`, `-`, `!`, and `*`.
```
Remove `*` from this list

Konicai commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
```
https://floodgate-uuid.heathmitchell1.repl.co/ works now so I would just include both

Konicai commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
+Alternatively, run the "/UUID" command in #botspam.
+Manual method:
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```
Can we remove this? There are 3 ways to get it automatically now so we are probably 3

onebeastchris replied:
I've changed it to just show how a floodgate uuid is "formatted" from a XUID - or should i rather remove it altogether?

Camotoy commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
+You can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
```
This wording, bluntly, sounds like a blog when we know what will happen. Every output of that site should need the conversion from below.

onebeastchris replied:
fixed in the latest commit

Konicai reviewed (changes requested):
I think my previous review got corrupted because there were changes

onebeastchris commented on src/main/resources/tags/links/uuid.tag:
```diff
@@ -2,5 +2,16 @@ type: text
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```
breh replied to wrong one
anyways, yes, I've added a new commit that mentions that this is velocity only vs bungee..which, ironically, the wiki doesn't do (no mention of velocity)
ig I'll make a wiki pr for that and just link to the wiki in this tag

Konicai replied:
oh lol alright

onebeastchris commented:
Okay, i've committed a few - hopefully more or less final changes. I removed the "*" prefix from the list of suggested prefixes, made https://github.com/GeyserMC/GeyserWiki/pull/108 and link to this wiki page in the !!bungee tag. And i've fixed "especially".. dunno why that one is a recurring spelling error on my side, sorry xD
